python/cudf/cudf/core/_base_index.py

```diff
@@ -626,10 +626,18 @@ def notna(self):
         """
         raise NotImplementedError
 
-    def to_pandas(self):
+    def to_pandas(self, nullable=False):
         """
         Convert to a Pandas Index.
 
+        Parameters
+        ----------
+        nullable : Boolean, Default False
+            If ``nullable`` is ``True``, the resulting series will be
+            having a corresponding nullable Pandas dtype. If ``nullable``
+            is ``False``, the resulting series will either convert null
+            values to ``np.nan`` or ``None`` depending on the dtype.
```

Comment:
```suggestion
            If ``nullable`` is ``True``, the resulting index will be
            having a corresponding nullable Pandas dtype. If ``nullable``
            is ``False``, the resulting index will either convert null
            values to ``np.nan`` or ``None`` depending on the dtype.
```

Reply:
Yikes, apologies for overlooking the text here.